### DISCUSSION TITLE
Question re: modal > `active`

### DISCUSSION THREAD

**Discussion by mitya33** [buefy/buefy#3523] [Q&A]
I'm not sure whether there's an issue with the docs for Modal or not.

The API lists the `active` param as the one that controls whether the modal is currently showing.

Yet none of the examples use this. Instead they have `v-model`, without explaining how that, instead of `active`, does the same thing. Am I missing something?

**Comment by jtommy** [comment]
Same thing

**Reply by mitya33** [reply]
OK, but I'd suggest that's far from clear to the more casual user. It's presumably not ideal for the API docs to say one thing and the examples to say another.
